database/migrations/2018_07_09_065431_insert_profileimagedriver_setting.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Could you remove this file and fix the issue like in this commit: 602b9f894116b44860e0a5249b8d1e34d7fa6944